### DISCUSSION TITLE
How do you trace variables?

### DISCUSSION THREAD

**Discussion by uahnbu** [kimlimjustin/xplorer#218] [Q&A] [answered]
I used `console.log` but cannot see the output in any VSCode pane. I had to use `alert` instead, which is kinda limited.

**Comment by kimlimjustin** [comment, marked as answer]
Xplorer has its devtool like browser does, if you run on Windows, simply type Ctrl + Shift + I will open it, if you run on macOS or Linux, it's quite tricky, you have to first comment the following line:
https://github.com/kimlimjustin/xplorer/blob/9d313d5be9956c9e408dc4879d17cc742c9cb2d8/src/Components/ContextMenu/contextMenu.ts#L109

and right-clicking on Xplorer will give you the default context menu, simply click on `Inspect` menu will open the devtool
